### DISCUSSION TITLE
Maplibre  - general information and instructions on the Maps SDK - Android

### DISCUSSION THREAD

**Discussion by tvaknin** [maplibre/maplibre-native#408] [Q&A]
Hi,

1. Is there any page for instructions on the Maps SDK for maplibre - Android?
the Mapbox instructions SDK page is not relevant, many functions are missing. 

2. How can I use the MapBox tiles with maplibre? e.g. the following won't work:

mapView?.getMapboxMap()?.loadStyleUri(Style.MAPBOX_STREETS)

Thanks

**Comment by wipfli** [comment]
Thanks for asking. I would be great if you could point out in an issue what exactly are the problems with the current documentation at https://maplibre.org/maplibre-gl-native/android/api/

Regarding tiles from mapbox, I think you will be best served by using their rendering libraries.

**Reply by tvaknin** [reply]
Thanks for your fast reply.
I didn't say there was a problem with the API but with the missing instructions.
I referred by this lib to the MapBox SDK instructions [https://docs.mapbox.com/android/maps/guides/install/](url) which I found not relevant to this lib.

I really miss some basic WIKI/get started on this GitHub page as follows:

Add the layout for the map:

            <com.mapbox.mapboxsdk.maps.MapView xmlns:mapbox="http://schemas.android.com/apk/res-auto"
                android:id="@+id/mapViewbox"
                android:layout_width="fill_parent"
                android:layout_height="250dp" />

Add a map:

mapView = findViewById(R.id.mapView)
.
.
.

**Reply by wipfli** [reply]
Oh yes I would love to have some tutorials to get started with native...

**Reply by tvaknin** [reply]
If this lib forked from MapBox, isn't the tutorials on MapBox should be pretty identical?
https://docs.mapbox.com/android/maps/guides/install/

If not, how should we know that if it's not documented? 

I think I miss the point of this lib on how it can be used on a native app.
